model_cards/bashar-talafha/multi-dialect-bert-base-arabic/README.md

```diff
@@ -0,0 +1,63 @@
+# Multi-dialect-Arabic-BERT
+This is a repository of Multi-dialect Arabic BERT model.
+
+By [Mawdoo3-AI](https://ai.mawdoo3.com/). 
+
+<p align="center">
+    <br>
+    <img src="https://github.com/mawdoo3/Multi-dialect-Arabic-BERT/blob/master/multidialct_arabic_bert.png" alt="Background reference: http://www.qfi.org/wp-content/uploads/2018/02/Qfi_Infographic_Mother-Language_Final.pdf" width="500"/>
```

Comment:
image link seems broken

Reply:
The link points to a private repo. I will make it public so that the picture will be shown.